package.json

```diff
@@ -190,8 +189,9 @@
     "webpack-stats-plugin": "^0.2.1"
   },
   "peerDependencies": {
-    "react": "0.14.x - 16.x",
-    "react-dom": "0.14.x - 16.x"
+    "react": "16.x",
+    "react-dom": "16.x",
```

Comment:
Have we considered the impact of dropping support for React 15?

Reply:
** This is a breaking change.
We are no longer supporting react 15 consumer apps